### PR TITLE
Fix for issue #6 and #8

### PR DESCRIPTION
**Pull request by MikalaiSivokha**
Fixed display message for Issue #6 

Modified user's data parsing logic for the cases when user'name starts with emoji, moved parsing logic to separate function, added unit tests for this function. Fix for Issue #8

### CODE REVIEW

File: plugins/top.py [hunk 1]
```diff
@@ -41,26 +41,32 @@ def handle_top_forward(message):
...
-            xp = r.group(5).strip()
-            fraction = re.findall(r'[^\w\s,]', name)
-            fraction = ''.join(fraction).strip("-, ")
-            name = name.replace(fraction, "")
-            if fraction in flags:
-                flag = flags[fraction]
-            else:
-                flag = 'deleted'
-            update_rating(name, position, flag, current_millis, level, xp)
+        for i in slices:
```
Comment (pavshka) [inline plugins/top.py:44]: please, use more descriptive variable name:
`raw_data` instead of `i`

File: plugins/top.py [hunk 2]
```diff
@@ -41,26 +41,32 @@ def handle_top_forward(message):
...
+    fraction = re.findall(r'[^\w\s,]', name)
+    if len(fraction) > 2:
+        fraction = fraction[:2]            
+    fraction = ''.join(fraction).strip("-, ")            
+    name = name.replace(fraction, "")
+    if fraction in flags:
+        flag = flags[fraction]
+    else:
+        flag = 'deleted'
+    return { 'name' : name, 'position': position, 'flag' : flag, 'level' : level, 'xp' : xp }
```
Comment (pavshka) [inline plugins/top.py:72]: Please, add new line

Comment (3mph4515) [reply plugins/top.py:72]: Really, please

File: plugins/top.py [hunk 3]
```diff
@@ -41,26 +41,32 @@ def handle_top_forward(message):
...
+            bot.send_message(message.chat.id, "Спасибо. Теперь можешь посмотреть общий рейтинг с помощью команды /top_от_до.\nВерхняя граница опциональный параметр")
         except Exception as e:
             print(e)
     else:
         try:
             bot.send_message(message.chat.id, "Принимаем только форварды /top из @ChatWarsBot")
         except Exception as e:
             print(e)
+
+def parse_player_data(raw_data):
```
Comment (pavshka) [inline plugins/top.py:57]: desperately needs refactoring

**Review by pavshka — APPROVED** [review]

**Review by 3mph4515 — APPROVED** [review]
LGTM

**Pull request merged**